ccjjmartin/aquifer-inope
# Challenge4 - Administrative interface

This PR seeks to:
- Create new permission for administrator reports.
- Allow administrators to see all feedback.
- Build an administrator report.
- Add new columns to table based on permissions and page.
- Sortable by new columns which required joins.
## Steps to test:
- Run update.php on existing sites.
- Verify you already have two policy nodes and two users (admin and staff)
- Create a few revisions of each node with submissions from both users
- Start with staff and verify that staff can only see their own submissions
  - Staff should also be able to edit their own submissions
  - Attempt to go to /admin/reports/inope
  - If you got an access denied as staff this is correct
- Now log in as the administrator
  - Go to /admin/reports/inope
  - Verify that the report contains additional columns called node and user
  - Verify that columns are sortable
  - Verify that default sort is by date
